### ISSUE TITLE
Kivent-Maps example 14 rendering with OpenGL error

### ISSUE THREAD

**Issue by itsthejoker** [kivy/kivent#183]
I have installed kivent-core + kivent_maps without issue and when running the included example, everything renders (and it's animating correctly) but the following error repeats on every frame update, filling up my console:

```[ERROR  ] OpenGL Error (Renderer._set_blend_func-preglBlendFunc) 1282 / 502 for None```

Any idea what this error is referring to? I'm having lots of difficulty tracking it down.

Complete pip freeze for the venv here:

```addict==2.0.0
appnope==0.1.0
codecov==2.0.5
coverage==4.3.1
Cython==0.25.2
decorator==4.0.10
freezegun==0.3.8
ipdb==0.10.1
ipython==5.1.0
ipython-genutils==0.1.0
KivEnt-Core==2.2.0.dev0
KivEnt-maps==1.0.0
KivEnt-particles==1.0.0
Kivy==1.9.1
Kivy-Garden==0.1.4
mock==2.0.0
numpy==1.11.3
pbr==1.10.0
pexpect==4.2.1
pickleshare==0.7.4
prompt-toolkit==1.0.9
ptyprocess==0.5.1
py==1.4.32
pygame==1.9.2
Pygments==2.1.3
pytest==3.0.5
pytest-cov==2.4.0
python-dateutil==2.6.0
requests==2.12.4
simplegeneric==0.8.1
six==1.10.0
tmx==1.8.1
traitlets==4.3.1
wcwidth==0.1.7
```

Also, kivent_core refuses to run without PyGame installed, saying that it cannot find a suitable window manager. Is this normal?

Thanks for your time!

Technical information: 
OSX Sierra 10.12.1 (16B2555)
Python 3.5.2
Virtualenv 15.0.3

**Comment by Kovak** [comment]
The 1282 error should be fixed on kivy master branch. 

You need to look into setting Kivy up with sdl2 instead of pygame to avoid using pygame. It does not look like you have the sdl2 deps installed.

**Comment by itsthejoker** [comment]
@Kovak thanks so much for the quick response. I've done as you suggested, but now I receive the error ```ValueError: kivy.graphics.vertex_instructions.Mesh has the wrong size, try recompiling. Expected 200, got 152``` when running. As this seems to be an issue with the Kivy master build (though I've recompiled with a few different options) I'll go ahead and close this as I believe the original question has been answered. Thanks for your help!

**Comment by Kovak** [comment]
You need to recompile kivent based on a newly compiled kivy

**Comment by itsthejoker** [comment]
I really appreciate your help here. When compiling kivent, I get the following error:

``` kivent_core/rendering/vertex_formats.pxd:1:0: 'kivy/graphics/c_opengl.pxd' not found``` and a bunch more errors related to the c_opengl folder not existing. AFAICT, it was removed in this kivy commit: https://github.com/kivy/kivy/commit/b38b9737fab9544882477f4d6438b24cbdc5144a

Any suggestions on how to move forward from here? I did make sure I'm working from the kivent master, if that helps.
